### DISCUSSION TITLE
Change podman storage folder

### DISCUSSION THREAD

**Discussion by antoba** [containers/podman#20507] [Q&A]
I installed "Podman Desktop" (on windows 10), now i need to move the folder in which podman save containers to another disk because it reached 50GB (!), i notice that the default is in "C:\Users\my_user\\.local\share\containers...", i want to move it to "D:\podman" (for.ex)
(For now i created a symbolik link)

This for more :

https://stackoverflow.com/questions/77330102/change-podman-storage-folder

Many thanks !

**Comment by yilmazdurmaz** [comment]
Hi @antoba, I was looking for this same info but the answer is either hard to find or does not exists. So, I used my WSL knowledge to create a possible one.

I answered on SO and I gave the steps to move a WSL disk, but it is true only if podman does not have coupled settings somewhere else.

I am not pro for podman but I use WSL images on my bigger disk a lot. On my PC, I created a default machine on C and moved to D this way. I ran the example podman without any problem.

So, try it now at your own risk, or read a bit more if there are some couplings created on first initialization.

Please give feedback later.

**Reply by antoba** [reply]
First of all thanks @yilmazdurmaz , but I was hoping for something simpler and aimed at podman, mostly here you are using wsl and not the configurability of Podman desktop itself. For now a symbolic link seems sufficient to me. I hope some podman expert can give an alternative.

Regards

**Reply by yilmazdurmaz** [reply]
@antoba it is natural I use WSL here because podman uses WSL. check this paragraph: https://podman.io/docs/installation#windows

podman for windows is merely an interface for the podman running in the WSL. by the "coupling" I was referring to firewall and access rights on the windows wide. otherwise, everything should be fine.

Anyways, I could not see a way to even create on another drive, let alone just move. In case I didn't miss it, we should ask for this feature imlemented: both create and move distro to other drives. 

PS: docker lets to create on another drive during the first installation. don't know if lets to move afterwards. so, podman should be able to do the same in theory. maybe later.

**Comment by zebratt** [comment]
@yilmazdurmaz Are you using wsl2?  "--import-in-place" is not valid at my situation,so i use  "wsl --export" and then "wsl --import" to move the podman-machine-default distribution to another driver, and finally i can use "wsl -d podman-machine-default" to login into that VM. The new problem comes, my windows 10 system can not communicate with the new location vm, podman looks like using ssh to communicate with the vm:
![image](https://github.com/containers/podman/assets/7269427/d9d7da8a-ae2b-4a07-bcb8-0fe1a5c27b68)

i have no idea how to establish a new connection between the new vm and the host.

**Reply by yilmazdurmaz** [reply]
> Are you using wsl2? 

Yes! And may I ask why you do not? If you are not restricted, I suggest you upgrade to it.

As for your problem, I think your approach is the faulty part here. You are assuming running its wsl machine would be enough.

you need to run the podman's VM with its own command: `podman machine start` (as hinted in that message, but not obvious for beginners).

And as an extra info, the reason we move wsl image with export/import (and topic of this post) is that its first command, `podman machine init`, does not have necessary coding to install to a custom location. export/import is just an extra step between `init` and `start` commands.
